tests/test_trezor.config.py

```diff
@@ -41,7 +41,7 @@ def test_wipe(self):
         self.assertEqual(v1, None)
 
     def test_lock(self):
-        for _ in range(128):
+        for i in range(128):
```

Comment:
i is not used, unnecessary change

Reply:
Agreed. I added that while debugging and forgot to clean up.